core/indigo-core/molecule/src/cml_loader.cpp

```diff
@@ -18,12 +18,11 @@
 
 #include "molecule/cml_loader.h"
 
-#include <tinyxml2.h>
-
 #include "base_cpp/scanner.h"
 #include "molecule/elements.h"
 #include "molecule/molecule.h"
 #include "molecule/molecule_scaffold_detection.h"
+#include <tinyxml2.h>
```

Comment:
That's incorrect change, we have three levels of headers, splitted by empty line:

1. standard library
2. third-party libraries
3. our code